smarts/core/mission_planner.py

```diff
@@ -182,3 +192,88 @@ def _edge_ids(self, pose: Pose, lane_id: str = None):
             edge_ids.append(next_edges[0].getID())
 
         return edge_ids
+
+    def uturn_waypoints(self, pose: Pose):
```

Comment:
This method is a little too low-level to be in the mission planner. Perhaps wrap in a helper we call out to.

Reply:
OK, I will add a TODO since this method is relying on the mission, road_network, waypoints.